x/concentrated-liquidity/pool.go

```diff
@@ -32,7 +36,7 @@ func (k Keeper) CreateNewConcentratedLiquidityPool(ctx sdk.Context, poolId uint6
 }
 
 // GetPool returns a pool with a given id.
-func (k Keeper) GetPool(ctx sdk.Context, poolId uint64) (types.PoolI, error) {
+func (k Keeper) GetPool(ctx sdk.Context, poolId uint64) (gammtypes.PoolI, error) {
```

Comment:
I don't think this should be returning a poolI from the Gamm types right?

Reply:
We will migrate pool interface to a better location in a future PR